src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/DeleteManyCommandResolver.java

```diff
@@ -35,6 +36,10 @@ public DeleteManyCommandResolver(OperationsConfig operationsConfig, ObjectMapper
 
   @Override
   public Operation resolveCommand(CommandContext commandContext, DeleteManyCommand command) {
+    // if there is no filter or is an empty filter, delete all the rows or truncate the table.
```

Comment:
"If there is no filter or filter is empty, use Truncate operation instead of Delete"?